additional_build_steps_royvanrijn.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+#  Copyright 2023 The original authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+source "$HOME/.sdkman/bin/sdkman-init.sh"
```

Comment:
I've just merged https://github.com/gunnarmorling/1brc/pull/301, which simplifies this a bit: no need for this additional script, its contents now just goes into the prepare script. Could you rebase + rework + squash this PR accordingly? Thx!

Reply:
Should work again now, followed the scripts @thomaswue used